### PR TITLE
Task #7 Method score() in class Game #21 with tests

### PR DESCRIPTION
**Pull request by anorberciak**

### CODE REVIEW

File: src/Game.cpp [hunk 1]
```diff
@@ -12,8 +12,12 @@ std::string Game::getName() const
 
 int Game::score() const
 {
-    //needs implementation after made convertValueToPairOfIntsFrame()
-    return 0;
+    int partialScore = 0;
+    for (int i=0; i<10; i++)
```
Comment (Tomek52) [inline src/Game.cpp]: @anorberciak you can write it in the correct formatting 
`for (int i = 0; i < 10; i++)`

File: src/Game.cpp [hunk 2]
```diff
@@ -12,8 +12,12 @@ std::string Game::getName() const
...
+    {
+        partialScore = partialScore + frame_[i].getValue();
```
Comment (Angan7a) [inline src/Game.cpp]: Here you can use short form `partialScore += frame _[i].getValue();`

File: src/Game.cpp (hunk 1 above)
Comment (anorberciak) [reply src/Game.cpp]: Done

File: src/Game.cpp (hunk 2 above)
Comment (anorberciak) [reply src/Game.cpp]: Done

**Pull request merged**